app/templates/components/build-header.hbs

```diff
@@ -183,6 +183,10 @@
           <SvgImage @name="icon-shared" @alt="Shared repository" @class="icon shared" /> Shared repository
         {{/if}}
       {{/if}}
+      {{#if this.build.priority}}
```

Comment:
@artursmirnov: 'priority' field is added by the backend team in the build api to calculate the highest priority build. Its value will be in boolean format.

Reply:
I thought it would be the case, but in order for this to work, this field should also be added to the `build` model as attribute. Otherwise it will never appear on records and will not be available here.

Reply:
Added the key in build model as well.

Reply:
Thank you 👍